src/components/Router.tsx

```diff
@@ -17,7 +17,7 @@ export const AssistedUiRouter: React.FC<{ allEnabledFeatures: FeatureListType }>
 );
 
 export const Router: React.FC<{ features: FeatureListType }> = ({ features, children }) => (
-  <>
+  <FeatureGateContextProvider features={features}>
```

Comment:
This will have to be moved as we're integrating with OCM

Reply:
Right. In general, I prefer to handle the context somewhere high in the hierarchy than copying it to each sub-component and dealing with passing of `features` props on multiple places. I can even imagine handling of the context provider wihtin OCM code-base. Let's consider it on particular code when it's available.